app/models/cake.rb

```diff
@@ -0,0 +1,5 @@
+class Cake < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true, length: { minimum: 3, maximum: 30 }
+  validates :kind, presence: true, inclusion: { in: %w(tasty non-gluten vegan), message: "%{value} is not a valid kind" }
```

Comment:
I didn't know it could be done that way. What is displayed when user does not select any value from select box?